miasm2/jitter/Jitllvm.c

```diff
@@ -6,6 +6,7 @@
 #include "queue.h"
 #include "vm_mngr.h"
 #include "vm_mngr_py.h"
+#include "bn.h"
```

Comment:
JitCore being already imported, is there a need for `bn.h`?

Reply:
Yes it is:
* `Jitcore.h` is needed by jitllvm (for jitcpu strcuts
* as jitcore.h export bn_t struct is need bn_t defnied BEFORE any inclusion
Conclusion:
bn.h is need in each C file using `jitcore.h`